controller/frontend/src/Controller/Frontend/Product/Standard.php

```diff
@@ -84,7 +84,7 @@ public function addFilterAttribute( \Aimeos\MW\Criteria\Iface $filter, array $at
 	public function addFilterCategory( \Aimeos\MW\Criteria\Iface $filter, $catId,
 		$level = \Aimeos\MW\Tree\Manager\Base::LEVEL_ONE, $sort = null, $direction = '+', $listtype = 'default' )
 	{
-		$catIds = ( !is_array( $catId ) ? explode( ',', $catId ) : $catId );
+		$catIds = ( !is_array( $catId ) ? array_unique( explode( ',', $catId ) ) : array_unique( $catId ) );
```

Comment:
Just a minor thing: If you move `array_unqiue()` to the front, you don't have to call it twice

Reply:
Actually, I don't call it twice, but I get your point. I submitted the final version with a single `array_unique()`. Feel free to edit anything if you want to change something else.